src/plugins/intel_cpu/src/nodes/non_max_suppression.h

```diff
@@ -4,82 +4,65 @@
 
 #pragma once
 
-#include <ie_common.h>
-#include <node.h>
-#include <string>
-#include <memory>
-#include <vector>
+#include "node.h"
+#include "kernels/x64/non_max_suppression.hpp"
 
-#define BOX_COORD_NUM 4
-
-using namespace InferenceEngine;
 
 namespace ov {
 namespace intel_cpu {
 namespace node {
 
-enum class NMSBoxEncodeType {
-    CORNER,
-    CENTER
-};
-
 enum NMSCandidateStatus {
     SUPPRESSED = 0,
     SELECTED = 1,
     UPDATED = 2
 };
 
-struct jit_nms_config_params {
-    NMSBoxEncodeType box_encode_type;
-    bool is_soft_suppressed_by_iou;
-};
-
-struct jit_nms_args {
-    const void* selected_boxes_coord[BOX_COORD_NUM];
-    size_t selected_boxes_num;
-    const void* candidate_box;
-    const void* iou_threshold;
-    void* candidate_status;
-    // for soft suppression, score *= scale * iou * iou;
-    const void* score_threshold;
-    const void* scale;
-    void* score;
-};
-
-struct jit_uni_nms_kernel {
-    void (*ker_)(const jit_nms_args *);
-
-    void operator()(const jit_nms_args *args) {
-        assert(ker_);
-        ker_(args);
+struct Point2D {
```

Comment:
This is to generic structure to keep it here. We either need to encapsulate it into NonMaxSuppression class or move to utils.

Reply:
Done